docs/notebooks/demo/navigator/text-to-code/navigator-data-designer-sdk-text-to-python.ipynb

```diff
@@ -0,0 +1,307 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 🎨 Navigator Data Designer SDK: Text-to-Python\n",
+    "\n",
+    "This notebook demonstrates how to use the Gretel Navigator SDK to create a synthetic data generation pipeline for Python code examples. We'll build a system that generates Python code based on natural language instructions, with varying complexity levels and industry focuses."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "%pip install -U gretel_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gretel_client.navigator import DataDesigner\n",
+    "from gretel_client.navigator.tasks.types import ValidatorType, EvaluationType, LLMJudgePromptTemplateType\n",
```

Comment:
You should be able to pass strings in place of the enum types. For example
```python
data_designer.add_evaluator(
    eval_type="text_to_python",  
    instruction_column_name="instruction",    
    response_column_name="code_implementation"
)
```
This way you don't have to worry about importing all of these.